backend/.gitignore

```diff
@@ -1,5 +1,6 @@
 # Byte-compiled / optimized / DLL files
 __pycache__/
+migrations/
```

Comment:
Why are migrations added here? Migrations are supposed to be pushed. Otherwise we can not build prod database